cpp/log/etcd_consistent_store_test.cc

```diff
@@ -289,8 +289,8 @@ TEST_F(EtcdConsistentStoreTest, TestSetServingSTHWontOverwriteWithOlder) {
   EXPECT_EQ(util::error::OUT_OF_RANGE, status.CanonicalCode()) << status;
 }
 
-
-TEST_F(EtcdConsistentStoreDeathTest, TestSetServingSTHChecksInconsistentSize) {
+TEST_F(EtcdConsistentStoreDeathTest,
+       TestSetServingSTHChecksInconsistentSizeDeathTest) {
```

Comment:
OK but I had consistent failures of the death tests OSX until I made this change. I wonder if I was reading out of date docs because the impression I got was that it applied to the tests. Also we might have some cases that mix non death tests?

I also note one test failed in the travis build but they all passed locally. Sigh.

I'll investigate all of this more tomorrow.